checker/tests/index/LubIndex.java

```diff
@@ -24,10 +23,8 @@ public static void Bottom(int @BottomVal [] arg, int @MinLen(4) [] arg2) {
         } else {
             arr = arg2;
         }
-        // :: error: (assignment.type.incompatible)
         int @MinLen(10) [] res = arr;
         int @MinLen(4) [] res2 = arr;
-        // :: error: (assignment.type.incompatible)
```

Comment:
These changes make sense, but they also alter the meaning of this test. Should we write a new test case for the merging of `@BottomVal` and `@MinLen(4)`?